doc/mapproxy_util_autoconfig.rst

```diff
@@ -26,8 +26,12 @@ Options
 
 .. cmdoption:: --capabilities <url|filename>
 
-  URL or filename of the WMS capabilities document. The tool will add `REQUEST` and `SERVICE` parameters to the URL as necessary.
+  URL or filename of the WMS capabilities document. The tool will add `REQUEST` and `SERVICE` parameters to the URL as necessary.  This option cannot be used with the --geopackage option.
```

Comment:
Can you create a separate pull request for the changes to autoconfig? It's unrelated to the GeoPackage cache backend and makes the pull request harder to review.